operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -72,6 +71,27 @@ spec:
 # Choose "anonymous" to allow full access to Kiali without requiring any credentials.
 # Choose "openshift" to use the OpenShift OAuth login which controls access based on
 # the individual's  RBAC roles in OpenShift. Not valid for non-OpenShift environments.
+# Choose "ldap" to enable LDAP based authentication. There are additional configurations for
+# under ldap in the yaml file. The configurations for LDAP are as follows
+#    ---
+#    strategy: ldap
+#    ldap:
+#       # AD IP host and port
+#       ldap_host: example.com
+#       ldap_port: 123
+#       ldap_use_ssl: false
+#       ldap_insecure_skip_verify: true
+#       ldap_user_filter: (cn=%s)
+#       ldap_group_filter: (cn=%s)
+#       ldap_attributes: ['memberOf', 'cn', 'mail']
+#       ldap_base: DC=example,DC=com
+#       ldap_bind_dn: CN={USERID},OU=xyz,OU=Users,OU=Accounts,DC=example,DC=com
+#       ldap_search_filter: (&(name={USERID}))
+#       # Time for token expiration
+#       ldap_token_expiration_min: 1440
```

Comment:
Token expiration setting is no longer used - delete this. In my suggestion in the earlier comment I removed it.

Reply:
Removed